.eslintrc.json

```diff
@@ -1,10 +1,17 @@
 {
   "plugins": ["node", "security", "react"],
-  "extends": ["airbnb", "plugin:security/recommended", "plugin:node/recommended"],
+  "extends": [
+    "airbnb",
+    "plugin:security/recommended",
+    "plugin:node/recommended"
+  ],
   "rules": {
-    "react/jsx-filename-extension": [1, {
-      "extensions": [".js"]
-    }],
+    "react/jsx-filename-extension": [
+      1,
```

Comment:
@Unesdala please fix your Vscode settings so that these types of auto formatting does not occur in the future, thanks